packages/pcd/webauthn-pcd/package.json

```diff
@@ -2,13 +2,28 @@
   "name": "@pcd/webauthn-pcd",
   "version": "0.10.0",
   "license": "GPL-3.0-or-later",
-  "main": "./index.ts",
-  "types": "./index.ts",
+  "main": "./dist/cjs/src/index.js",
+  "module": "./dist/esm/src/index.js",
+  "types": "./dist/types/src/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/types/src/index.d.ts",
+      "import": "./dist/esm/src/index.js",
+      "require": "./dist/cjs/src/index.js"
+    }
+  },
+  "files": [
+    "dist",
+    "./README.md",
+    "./LICENSE"
+  ],
```

Comment:
Ah, this is because I fixed it in [this PR](https://github.com/proofcarryingdata/zupass/pull/1404) and then rebased on top of `main`, having already fixed it in this branch too.